rclcpp/test/topic_statistics/test_subscription_topic_statistics.cpp

```diff
@@ -63,6 +77,42 @@ class TestSubscriptionTopicStatistics : public SubscriptionTopicStatistics<Callb
   }
 };
 
+/**
+ * Empty publisher node: used to publish empty messages
+ */
+class EmptyPublisher : public rclcpp::Node
```

Comment:
This should also either be `final` or have a virtual destructor I think.

Reply:
Will add a virtual default dtor.